style.css

```diff
@@ -0,0 +1,100 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  width: 375px;
+}
+
+header {
+  background-color: #3c3a39;
+  height: 48px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-left: 2%;
+  padding-right: 2%;
+}
+
+.logo-placeholder {
+  padding: 10px 12px 10px 12px;
+}
+
+.logo-placeholder a {
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 24px;
+  text-align: center;
+  color: #fff;
+}
+
+.headlines {
+  height: 580px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background: #1c1a19;
+  color: #fff;
+  background-image: url(./assets/Group.png);
+  background-repeat: no-repeat;
+  background-position: 100% 115%;
+}
+
+.primary-text {
+  font-family: "Crete Round", sans-serif;
+  font-size: 56px;
+  font-weight: 400;
+  line-height: 64px;
+  letter-spacing: 0.3px;
+  text-align: left;
+  width: 327px;
+  color: #ff6b00;
+}
+
+.supporting-text {
+  margin-top: -10%;
+  width: 327px;
+  font-family: "Roboto", sans-serif;
+  font-size: 24px;
+  font-weight: 700;
+  line-height: 32px;
+  text-align: left;
+}
```

Comment:
- [OPTIONAL] Kindly make sure to change the color of this text in order to match the provided Figma design.

  - ###YOUR DESIGN
  
  ![image](https://user-images.githubusercontent.com/87197412/227444786-5f9d449a-4d14-48e4-bd78-eac88cfd0eff.png)

  - ### FIGMA DESIGN
  
   ![image](https://user-images.githubusercontent.com/87197412/227444919-0110f4e6-2f2b-428a-8545-95b9b4d384b5.png)

Reply:
Thank you